Documentation/cmdref/cilium-agent.md

```diff
@@ -387,7 +387,7 @@ cilium-agent [flags]
       --tofqdns-pre-cache string                                  DNS cache data at this path is preloaded on agent startup
       --tofqdns-proxy-port int                                    Global port on which the in-agent DNS proxy should listen. Default 0 is a OS-assigned port.
       --tofqdns-proxy-response-max-delay duration                 The maximum time the DNS proxy holds an allowed DNS response before sending it along. Responses are sent as soon as the datapath is updated with the new IP information. (default 100ms)
-      --trace-payloadlen int                                      Length of payload to capture when tracing (default 128)
+      --trace-payloadlen int                                      Length of payload to capture when tracing packets. (default 128)
```

Comment:
Is this auto aligned to the cache line size? should we mention that or otherwise specify that it needs to be that.